shell/platform/darwin/ios/framework/Source/SemanticsObjectTest.mm

```diff
@@ -100,6 +100,29 @@ - (void)testPlainSemanticsObjectWithLabelHasStaticTextTrait {
   XCTAssertEqual([object accessibilityTraits], UIAccessibilityTraitStaticText);
 }
 
+- (void)testNodeWithImplicitScrollIsAnAccessibilityElementWhenItisHidden {
+  fml::WeakPtrFactory<flutter::AccessibilityBridgeIos> factory(
+      new flutter::MockAccessibilityBridge());
+  fml::WeakPtr<flutter::AccessibilityBridgeIos> bridge = factory.GetWeakPtr();
+  flutter::SemanticsNode node;
+  node.flags = static_cast<int32_t>(flutter::SemanticsFlags::kHasImplicitScrolling) |
```

Comment:
Are these casts necessary?  They should already be int32_t's.  (fwiw I think people use unsigned ints for flags to avoid 2's compliment weirdness)

Reply:
I got this error if i get rid of the static cast
```
error: invalid operands to binary expression ('flutter::SemanticsFlags' and 'flutter::SemanticsFlags')
  node.flags = flutter::SemanticsFlags::kHasImplicitScrolling |
```

Reply:
weird, thanks